Sources/Main/CLIUtility.swift

```diff
@@ -21,5 +21,9 @@ public class CLIUtility {
         return parsedArguments.get(port) ?? 5000
     }
 
+    public func getDirectory() -> String {
+        return parsedArguments.get(directory) ?? "./cob_spec/public"
```

Comment:
I like that you have a default value here, but what happens if `./cob_spec/public` doesn't exist? It seems you're just trusting that they will either enter a value or the default must be there.

This seems like a place you may want to 'fail fast', i.e. `Error: No Public Directory Found`